server/store/daily_summary_store.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package store
+
+import (
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/kvstore"
+)
+
+type DailySummaryStore interface {
+	LoadDailySummaryIndex() (DailySummaryIndex, error)
+	SaveDailySummaryIndex(dsumIndex DailySummaryIndex) error
+}
+
+type DailySummarySettings struct {
+	MattermostUserID string `json:"mm_id"`
+	Enable           bool   `json:"enable"`
+	PostTime         string `json:"post_time"` // Kitchen format, i.e. 8:30AM
```

Comment:
I would add a LastPostTime field to avoid repetition over different clusters on HA. It is not optimal solution, but may help.